spec/models/organization_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Organization, type: :model do
+  describe 'associations' do
+    it 'has many tradings' do
+      association = described_class.reflect_on_association(:tradings)
+      expect(association.macro).to eq(:has_many)
+    end
+  end
+
+  describe 'validations' do
+    it 'validates presence of name' do
+      organization = Organization.new
+      expect(organization).not_to be_valid
+    end
+
+    it 'validates uniqueness of name' do
+      Organization.create(name: 'Test Organization')
+      organization = Organization.new(name: 'Test Organization')
+      expect(organization).not_to be_valid
```

Comment:
Query: why its invalid?

Reply:
Bhaiya, Organization name is a unique identity and two organization can't have the exact same name, like there can't be two Apple Inc. as in the previous line, an organization named 'Test Organization' already has already been created, the second one became invalid